space-plugins/nuxt-base/server/middleware/02.app_bridge.global.ts

```diff
@@ -1,3 +1,5 @@
+import { APP_BRIDGE_TOKEN_HEADER_KEY } from '../../utils/const';
```

Comment:
I had to explicitly import this, because `nuxt-starter` project failed to auto-import this. I guess the same auto import configuration should be at nuxt-starter to make it work without this import. But we can revisit that later.